src/Standards/Squiz/Sniffs/Commenting/FunctionCommentThrowTagSniff.php

```diff
@@ -160,21 +168,23 @@ protected function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScop
         // Only need one @throws tag for each type of exception thrown.
         $thrownExceptions = array_unique($thrownExceptions);
 
-        $throwTags    = array();
-        $commentStart = $tokens[$commentEnd]['comment_opener'];
-        foreach ($tokens[$commentStart]['comment_tags'] as $tag) {
-            if ($tokens[$tag]['content'] !== '@throws') {
-                continue;
-            }
-
-            if ($tokens[($tag + 2)]['code'] === T_DOC_COMMENT_STRING) {
-                $exception = $tokens[($tag + 2)]['content'];
-                $space     = strpos($exception, ' ');
-                if ($space !== false) {
-                    $exception = substr($exception, 0, $space);
+        $throwTags = array();
+        if ($commentEnd < $currScope) {
```

Comment:
Above condition is added - the condition is satisfied always when we skip functions without doc comment and if false when we process functions without doc comment. There is no any other changes, just added one more indent.